src/styles/index.scss

```diff
@@ -1,3 +1,245 @@
+:root {
+  --light-blue: #00acdc;
+  --column-count: 4;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
You have a global style for box-sizing. This is not a good practice, because it affects all elements and can lead to unexpected behavior. You should apply box-sizing only to the elements that need it.